src/components/split_button/_split_button_control.scss

```diff
@@ -0,0 +1,68 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+// sass-lint:disable no-vendor-prefixes
+
+$splitButtonSeparatorColor: shade($ouiColorPrimary, 50%);
+
+.ouiSplitButton {
+
+  // Remove individual button style
+  .ouiButton,
+  .ouiButtonIcon {
+    border-width: 0px;
+    border-top-right-radius: 0px;
+    border-top-left-radius: 0px;
+    border-bottom-right-radius: 0px;
+    border-bottom-left-radius: 0px;
+
+    &:hover,
+    &:active,
+    &:focus {
+      -webkit-transform: none;
+      transform: none;
+    }
+  }
+
+  .ouiSplitButtonControl {
+
+    border: 1px solid $ouiColorPrimary;
+    border-top-right-radius: 4px;
+    border-top-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-bottom-left-radius: 4px;
+
+    // Animate wrapper element only when primary button activated
+    &:has(.ouiSplitButtonControl--primary:hover:not([class*='isDisabled'])) {
+      -webkit-transform: translateY(-1px);
+      transform: translateY(-1px);
+    }
+
+    &:has(.ouiSplitButtonControl--primary:active:not([class*='isDisabled'])) {
+      -webkit-transform: translateY(1px);
+      transform: translateY(1px);
+    }
+
+    &:has(.ouiSplitButtonControl--primary:focus:not([class*='isDisabled'])) {
+      animation: ouiButtonActive $ouiAnimSpeedNormal $ouiAnimSlightBounce;
+    }
+
+    .ouiSplitButtonControl--primary {
+      border-right: 1px solid $splitButtonSeparatorColor;
+
+    }
+
+  }
+}
+
+.ouiSplitButtonControl {
+  display: block;
+}
```

Comment:
I don't think this is needed right? The element is already `div`, so `display` should be `block` by default